LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -503,6 +532,7 @@ void updateTelemetry() {
   sprintf(telemetry_buff + 50, "%02d", gps.satellites.isValid() ? (int)gps.satellites.value() : 0);
   telemetry_buff[52] = 'S';
   telemetry_buff[53] = ' ';
+  telemetry_buff[54] = 
```

Comment:
this got added inadvertently again.  could you please remove?

Reply:
:) that was in my branch when tyler pulled